static/js/main.js

```diff
@@ -42,13 +42,47 @@ $('#CollapseToggleHide').click(function(){
 });
 
 
-//Implementation of Light Mode. Could be handled in a more modern way but it does it's job.
+//Themes. Could be handled in a more modern way but it does it's job.
+$oldbody = null;
+$oldtext = null
+
 $('#ModeToggle').change(function(){
-    $(document.body).toggleClass('LightModeBody');
-    $('.textbox').toggleClass('LightModeTextbox');
-    $('.searchbar').toggleClass('LightModeTextbox');
-    $('.widget').toggleClass('LightModeTextbox');
-    $('.header_header').toggleClass('LightModeTextbox');
+
+    $theme = $(this).val();
+
+    if ($theme == "classic" || $oldbody){
+        $(document.body).removeClass($oldbody);
+        $('.textbox').removeClass($oldtext);
+        $('.searchbar').removeClass($oldtext);
+        $('.widget').removeClass($oldtext);
+        $('.header_header').removeClass($oldtext);
+    }
+
+    if (!($theme == "classic")){
+
+        switch($theme){

```

Comment:
for each theme, theres a case containing the setters for `$themebody` and `$themetextbox` according to `main.css`